test/contrib/automotive/gm/gmlanutils.uts

```diff
@@ -95,14 +95,12 @@ s.close()
 = Check if can-isotp and can-utils are installed on this system
 ~ linux
 p = subprocess.Popen('lsmod | grep "^can_isotp"', stdout = subprocess.PIPE, shell=True)
-if p.wait() == 0:
-    if b"can_isotp" in p.stdout.read():
-        p = subprocess.Popen("isotpsend -s1 -d0 %s" % iface0, stdin = subprocess.PIPE, shell=True)
-        p.stdin.write(b"01")
-        p.stdin.close()
-        r = p.wait()
-        if r == 0:
-            ISOTP_KERNEL_MODULE_AVAILABLE = True
+if p.wait() == 0 and b"can_isotp" in p.stdout.read():
+    p = subprocess.Popen("isotpsend -s1 -d0 %s" % iface0, stdin = subprocess.PIPE, shell=True)
+    p.stdin.write(b"01")
```

Comment:
If the process fails to open and the shell closes too quickly, trying to write on stdin will make a raise a broken pipe exception.

I'm currently cleaning up the test suite and fixing all DeprecationWarnings, [it fails precisely on this test](https://travis-ci.com/secdev/scapy/builds/151041183) (if one of you could have a look it would be great..)

Not sure why this happens. Note i removed the `trusty` dependency (because travis will soon stop providing it). might be the root cause

Reply:
In my case i fixed it using https://github.com/secdev/scapy/pull/2478/commits/7091d4cae838ca9c0c6860d7659bea6f4e5ba8cf

Reply:
@gpotter2 Please see https://github.com/secdev/scapy/pull/2459/commits/9a841361d24157e6cf2c9ba064a344c0dc6f2c9b for the requested changes